### DISCUSSION TITLE
Change sidebar menu scrollbar style

### DISCUSSION THREAD

**Discussion by aymanalareqi** [filamentphp/filament#4116] [Ideas]
when the sidebar have many navigation items the native scrollbar appear like this
![scrollbar](https://user-images.githubusercontent.com/77433449/191363335-0a13f089-73f5-4396-b657-d01d34d8a0f2.PNG)
it could be better to change the scrollbar style instead of native scrollbar
we could use something like this 
https://www.npmjs.com/package/tailwind-scrollbar

**Comment by studiowizjo** [comment]
I thought about PR for this for some time, but didn't inspected yet possibilities in Tailwind. It is only few lines in css, so maybe additional plugin will be too much overhead.

**Reply by danharrin** [reply]
https://github.com/filamentphp/filament/pull/3764#issuecomment-1232718136

**Reply by aymanalareqi** [reply]
thanks @danharrin 
but what if we could use library like this 
https://perfectscrollbar.com
which could solve browsers styling issues

**Comment by aliloubm** [comment]
.filament-sidebar-nav::-webkit-scrollbar {
    width: 6px;
    background-color: #fff;
}
/* Customize Thumb */
.filament-sidebar-nav::-webkit-scrollbar-thumb {
    background-color: #cccccc;
    border-radius: 10px;
}
.filament-sidebar-nav::-webkit-scrollbar-thumb:hover {
    background-color: #cccccc;
}
.filament-sidebar-nav::-webkit-scrollbar-thumb:active {
    background-color: #cccccc;
}
